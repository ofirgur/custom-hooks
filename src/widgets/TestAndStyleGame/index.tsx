import React from 'react';
import { INITIAL_GAME_STATE } from './utils/constants';
import { GameStateContextProvider } from './utils/context';
import GameOutletScreen from './components/GameOutletScreen';

const TestAndStyleGame = () => {
  return (
    <GameStateContextProvider initialState={INITIAL_GAME_STATE}>
      <GameOutletScreen />
    </GameStateContextProvider>
  );
};

export default TestAndStyleGame;
