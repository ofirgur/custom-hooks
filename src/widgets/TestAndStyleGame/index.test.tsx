import * as React from 'react';
import { render } from '@testing-library/react';
import TestAndStyleGame from './index';

describe('TestAndStyleGame', () => {
  it('should render correctly', () => {
    const { container } = render(<TestAndStyleGame />);
    expect(container).toMatchSnapshot();
  });
});
