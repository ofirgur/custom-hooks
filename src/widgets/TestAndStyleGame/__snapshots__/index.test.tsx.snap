// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TestAndStyleGame should render correctly 1`] = `
<div>
  <div
    class="css-1e2fy0l"
  >
    <h1
      class="css-11919ur"
    >
      Posts List
    </h1>
  </div>
</div>
`;
